libs/@hashintel/design-system/src/fa-icons/fa-calendar-clock-regular.tsx

```diff
@@ -0,0 +1,9 @@
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons";
```

Comment:
At some point it'd be nice to be more consistent with how we add FA icons, appreciate this is approach makes sense here because of the implementation in `data-types-options-context.tsx`

Reply:
Definitely agree